frontend/app/services/readonly-ceqr-data-store.js

```diff
@@ -1,6 +1,6 @@
 import Service from '@ember/service';
 import { inject as service } from '@ember/service';
-import { fetchAndSaveModalSplit } from 'labs-ceqr/utils/modal-split';
+import { fetchAndSaveModalSplit } from 'labs-ceqr/utils/modalSplit';
```

Comment:
Is the convention to camelcase non-ember js files?

Reply:
oh idk why i did this...